source/aggregation-pipeline-builder.txt

```diff
@@ -112,35 +112,31 @@ your chosen view mode:
       .. procedure:: 
          :style: normal
 
-         .. step:: In the aggregation pipeline pane, ensure the 
+         .. step:: Select the :guilabel:`Stage` view.
+            
+            In the aggregation pipeline pane, ensure the 
             :guilabel:`{} Stages` toggle switch is selected.
 
             .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
                :alt: Toggle on stage view mode 
 
-         .. step:: At the bottom of the aggregation pipeline pane, select the 
-            :guilabel:`+ Add Stage` button to open a new aggregation stage 
-            modal.
+         .. step:: Add an aggregation stage. 
+            
+            At the bottom of the aggregation pipeline pane, click the 
+            :guilabel:`+ Add Stage` button.
+            
+         .. step:: Select an aggregation pipeline stage.
             
-         .. step:: At the lower-left corner of the aggregation pipeline pane, 
+            On the upper-left corner of the aggregation stage card, 
             click the :guilabel:`Select...` dropdown and select the 
             :manual:`aggregation pipeline stage 
             </reference/operator/aggregation-pipeline/>` to use for the first 
             stage of the pipeline. 
 
-         .. step:: Fill in your selected pipeline stage. 
+         .. step:: Fill in your pipeline stage. 
             
-            If you enable :guilabel:`Preview`, preview documents update 
-            automatically to the right of the stage to reflect the results of 
-            your pipeline as it progresses. You can adjust the width of the 
+            Fill in your selected stage. You can adjust the width of the 
             pipeline stage by dragging its border to the right.
-            
-            .. figure:: /images/compass/agg-builder-match-stage.png 
-               :alt: Aggregation Builder match stage example
-         
-         .. step:: Click :guilabel:`Add Stage` to add additional aggregation 
-            stages below your most recently added aggregation stage. Repeat 
-            steps 1 and 2 for each additional stage.
 
             .. note::
                
```

Comment:
[suggestion, non-blocking] The screenshot in this include is really difficult to read. You might consider taking a more focused screenshot of the most relevant section of the page.